MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/proc_revoke-api-key-admin.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: PROCEDURE

[id="revoke-api-key-admin_{context}"]
= Revoke API keys as an administrator

[role="_abstract"]

> Administrators can revoke an application programming interface (API) key before its lifespan in the migration toolkit for application (MTA) web console.

Reviewer: ```suggestion
As an administrator, you can revoke an application programming interface (API) key before it expires in the {ProductFirstRef} web console.
```

Author: Please let me check with the SME if we can use expires here. API keys have a separate parameter called expiration, other than lifespan.

.Prerequisites

You installed {ProductShortName} 8.2.0 in a Red Hat OpenShift cluster.

.Procedure

> . Log in to the {ProductShortName} web console.
> . In the Administration mode, go to *User Management*.
> . Select *Tokens*.
> . On the *Tokens* page, click the options icon.
> . Click *Revoke*.

Reviewer: ```suggestion
. Log in to the {ProductShortName} web console.
. In the *Administration* mode, click *User Management*.
. Click *Tokens*.
. On the *Tokens* page, click the options icon.
. Click *Revoke*.
```
